apps/web/components/eventtype/CreateEventType.tsx

```diff
@@ -51,7 +52,7 @@ interface CreateEventTypeBtnProps {
 export default function CreateEventTypeButton(props: CreateEventTypeBtnProps) {
   const { t } = useLocale();
```

Comment:
@hariombalhara why not use this instead?

```suggestion
  const { t, isLocaleReady } = useLocale();
```